contrib/libvirt/template.py

```diff
@@ -0,0 +1,126 @@
+pkgname = "libvirt"
+pkgver = "10.2.0"
+pkgrel = 0
+build_style = "meson"
+configure_args = [
+    "-Dattr=enabled",
+    "-Dblkid=enabled",
+    "-Dcapng=enabled",
+    "-Dcurl=enabled",
+    "-Ddriver_qemu=enabled",
+    "-Ddriver_secrets=enabled",
+    "-Dfirewalld=enabled",
+    "-Dfirewalld_zone=enabled",
+    "-Dfuse=enabled",
+    "-Dlibiscsi=enabled",
+    "-Dlibnl=enabled",
+    "-Dlibssh2=enabled",
+    "-Dno_git=true",
+    "-Dnumactl=enabled",
+    "-Dpackager=Chimera Linux",
+    "-Dpciaccess=enabled",
+    "-Dstorage_dir=enabled",
+    "-Dstorage_disk=enabled",
+    "-Dstorage_fs=enabled",
+    "-Dstorage_lvm=enabled",
+    "-Dstorage_mpath=enabled",
+    "-Dstorage_scsi=enabled",
+    "-Dstorage_zfs=enabled",
+    "-Dudev=enabled",
+    "-Dyajl=enabled",
+    f"-Dpackager_version={pkgver}",
+    "-Dqemu_user=virtqemu",
+    "-Dqemu_group=virtqemu",
+]
+hostmakedepends = [
+    "libxml2-progs",
+    "lvm2",  # buildtime check
+    "meson",
+    "mkfs",  # buildtime check
+    "mount",  # buildtime check
+    "perl",
+    "pkgconf",
+    "python-docutils",
+    "xsltproc",
+]
+makedepends = [
+    "acl-devel",
+    "attr-devel",
+    "bash-completion",
+    "device-mapper-devel",
+    "fuse-devel",
+    "glib-devel",
+    "gnutls-devel",
+    "libcap-ng-devel",
+    "libcurl-devel",
+    "libiscsi-devel",
+    "libnl-devel",
+    "libnuma-devel",
+    "libpcap-devel",
+    "libpciaccess-devel",
+    "libsasl-devel",
+    "libssh-devel",
+    "libssh2-devel",
+    "libtirpc-devel",
+    "libxml2-devel",
+    "linux-headers",
+    "parted-devel",
+    "polkit-devel",
+    "readline-devel",
+    "udev-devel",
+    "yajl-devel",
+]
+checkdepends = [
+    "pahole",
+    "python-black",
+    "python-flake8",
+    "python-pytest",
+]
+depends = [
+    "dnsmasq",
+]
+pkgdesc = "API, daemon, and management tool for virtualization"
+maintainer = "cesorious <cesorious@gmail.com>"
+license = "LGPL-2.1-only"
+url = "https://libvirt.org"
+source = f"https://download.libvirt.org/libvirt-{pkgver}.tar.xz"
+sha256 = "215772bc5dc4a672e67ffa9de3774f05ed4b7ed282dbe296ec5c9fec01dd7ae3"
+options = ["!lto"]
```

Comment:
comment why

Reply:
leftover from Nova's initial template, seems to work fine with lto actually